Text to Speech/text_to_speech.py

```diff
@@ -0,0 +1,7 @@
+from gtts import gTTS
+import os
+text = 'Welcome to GirlScript Summer of Code!'
+language = 'en'
+myobj = gTTS(text=text, lang=language, slow=False)
+myobj.save("audio.mp3")
+os.system("mpg321 audio.mp3")
```

Comment:
Hmm, seems a simple one. Anyway I'll open an issue regarding integrating this to the site itself